python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_graphs.py

```diff
@@ -50,11 +50,11 @@ def drawFamily(factory, degreeMax=5):
 # Draw the 5-th first members of the Jacobi family.
 
 # %%
-# Create the Jacobi polynomials family using the default Jacobi.ANALYSIS
+# Create the Jacobi polynomials family using the default analysis  
 # parameter set
```

Comment:
The reader should understand that we're choosing one of two available parametrizations for the Beta distribution. It might be helpful to create a link to the API doc page of `JacobiFactory`, which contains this explanation.

Reply:
I think its okay to merge as is because it fulfills the initial goal.
Imho no need to add a link to the API, it will be tedious to add these for each example.

Reply:
The correction is somewhat worse than the original, but cryptic, wording, because the word "analysis" is unrelated to the content of the script. At this point, we must explain that the `JacobiFactory` has two different parametrizations. An additional bug is that:

https://github.com/openturns/openturns/blob/9adf87d2687d5dc956214bb66326dac0688d69b8/python/src/JacobiFactory_doc.i.in#L25

does not state which of either ot.JacobiFactory.ANALYSIS or ot.JacobiFactory.PROBABILITY is the default setting.